bin/ck8s-kubespray

```diff
@@ -25,6 +25,11 @@ usage() {
     echo "      args: <prefix> [<options>]" 1>&2
     echo "  upgrade                                     prepares config for upgrade" 1>&2
     echo "      args: <wc|sc|both> <version> prepare" 1>&2
+    echo "      args: <version> prepare" 1>&2
```

Comment:
Is there a reason why this line was added?

Reply:
Not really, I did a rebase and that was the source of the conflict, so I kept it to not override any changes.
I guess as you've worked on splitting the upgrade, I suppose that line is no longer needed, right ?

Reply:
That makes sense, yes it should be fine to remove :slightly_smiling_face: